### DISCUSSION TITLE
How to add readme to profile

### DISCUSSION THREAD

**Discussion by Lazar445** [community/community#64632] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How to add readme to profile?

**Comment by Mr-Proxy-source** [comment]
1. In the upper-right corner of any page, use the  drop-down menu, and select New repository.
2. Under "Repository name", type a repository name that matches your GitHub username. For example, if your username is "octocat", the repository name must be "octocat".
3. Optionally, in the "Description" field, type a description of your repository. For example, "My personal repository."
4. Select Public.
5. Select Initialize this repository with a README.
6. Click Create repository.
7. Above the right sidebar, click Edit README.
8. The generated README file is pre-populated with a template to give you some inspiration for your profile README.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
